build_runner/lib/src/entrypoint/test.dart

```diff
@@ -135,7 +136,9 @@ class TestCommand extends BuildRunnerCommand {
           precompiledPath,
         ]..addAll(extraTestArgs),
         mode: ProcessStartMode.inheritStdio);
-    return testProcess.exitCode;
+    _ensureProcessExit(testProcess);
+    final testExitCode = await testProcess.exitCode;
+    return testExitCode;
```

Comment:
nit: you can just `return testProcess.exitCode;` here.

Reply:
whoops, removed some code I had here previously. Thanks :)